app/submitter/converter.py

```diff
@@ -88,7 +88,13 @@ def prepare_answers(metadata_store, questionnaire, answers):
             if item is not None:
                 value = answers[key]
                 if value is not None:
-                    data[item.code] = Converter._encode_value(value)
+                    if item.code not in data:
```

Comment:
Had to do something or you get an error when you try to submit. Happy to have a discussion about what the downstream format should/needs to be.